mmpose/datasets/datasets/body/mpii_trb_dataset.py

```diff
@@ -122,39 +123,29 @@ def _load_annotations(self) -> List[dict]:
             center = np.array(ann['center'], dtype=np.float32)
             scale = np.array([ann['scale'], ann['scale']],
                              dtype=np.float32) * pixel_std
```

Comment:
```suggestion
            # center, scale in shape [1, 2] and bbox in [1, 4]
            center = np.array([ann['center']], dtype=np.float32)
            scale = np.array([[ann['scale'], ann['scale']]],
                             dtype=np.float32) * pixel_std
```
`center` and `scale` should have shape [1, 2]